sdk/src/client/node_api/core/routes.rs

```diff
@@ -349,6 +348,56 @@ impl ClientInner {
             .await
     }
 
+    // Commitments routes.
+
+    /// Gets the slot commitment by the given slot commitment id.
+    /// GET /api/core/v3/commitments/{commitmentId}
+    pub async fn get_slot_commitment_by_id(&self, slot_commitment_id: &SlotCommitmentId) -> Result<SlotCommitment> {
+        let path = &format!("api/core/v3/commitments/{slot_commitment_id}");
+
+        self.node_manager
+            .read()
+            .await
+            .get_request::<SlotCommitment>(path, None, self.get_timeout().await, false, true)
+            .await
+    }
+
+    /// Gets the slot commitment, as raw bytes, by the given slot commitment id.
+    /// GET /api/core/v3/commitments/{commitmentId}
+    pub async fn get_slot_commitment_by_id_raw(&self, slot_commitment_id: &SlotCommitmentId) -> Result<Vec<u8>> {
+        let path = &format!("api/core/v3/commitments/{slot_commitment_id}");
+
+        self.node_manager
+            .read()
+            .await
+            .get_request_bytes(path, None, self.get_timeout().await)
+            .await
+    }
+
+    /// Gets the slot commitment by the given slot index.
+    /// GET /api/core/v3/commitments/by-index/{index}
+    pub async fn get_slot_commitment_by_index(&self, slot_index: &SlotIndex) -> Result<SlotCommitment> {
+        let path = &format!("api/core/v3/commitments/by-index/{slot_index}");
+
+        self.node_manager
+            .read()
+            .await
+            .get_request::<SlotCommitment>(path, None, self.get_timeout().await, false, true)
+            .await
+    }
+
+    /// Gets the slot commitment, as raw bytes, by the given slot index.
+    /// /api/core/v3/commitments/by-index/{index}
```

Comment:
totally nit-picky...so ignore if you want ... but here you missed the `GET`, also you use `index` in the doc, but `slot_index` would be consistent.

Reply:
Done